src/main/java/com/starrocks/connector/spark/rest/models/Schema.java

```diff
@@ -51,8 +53,23 @@ public void setProperties(List<Field> properties) {
         this.properties = properties;
     }
 
-    public void put(String name, String type, String comment, int scale, int precision) {
-        properties.add(new Field(name, type, comment, scale, precision));
+
+    public void setTableId(Long tableId) {
+        this.tableId = tableId;
+    }
+
+    public Long getTableId() {
+        return tableId;
+    }
+
+    public void put(String name,
```

Comment:
How about constructing the field outside, and make the `put` interface simple and scalable?